<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TacticMapWindow</class>
 <widget class="QMainWindow" name="TacticMapWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="PwGisWidget" name="pwgiswidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnOpenMap">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Открыть карту</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddMarker">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Маркер</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddCircle">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Круг</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddPath">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Путь</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddPolygon">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Полигон</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRect">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Прямоугольник</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddSlice">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Сектор</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxStyle">
        <item>
         <property name="text">
          <string>Добавить\удалить</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Добавить\удалить (по имени)</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxStyle">
        <property name="text">
         <string>Стиль1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxStyle2">
        <property name="text">
         <string>Стиль2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxStyle3">
        <property name="text">
         <string>Стиль3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancelStyle">
        <property name="text">
         <string>Отменить стиль</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnCrazy">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Crazy</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PwGisWidget</class>
   <extends>QWidget</extends>
   <header location="global">PwGis/pwgiswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
